src/test/java/util/xcmailr/XcMailrApiTest.java

```diff
@@ -25,157 +16,87 @@
-                                                                                                     : "", System.getenv("EMAIL_LOGIN") != null ? System.getenv("EMAIL_LOGIN")
-                                                                                                                                                : "", System.getenv("EMAIL_PASSWORD") != null ? System.getenv("EMAIL_PASSWORD")
-                                                                                                                                                                                              : "", System.getenv("EMAIL_SERVER") != null ? System.getenv("EMAIL_SERVER")
-                                                                                                                                                                                                                                          : "", 25, false, true);
+    protected final String tempEmail = "testTest1@varmail.net";
 
     @Before
     public void configureApiToken()
     {
         final String apiToken = System.getenv("XCMAILR_TOKEN");
         if (apiToken != null)
         {
-            properties2.put("xcmailr.apiToken", apiToken);
+            writeProperty("xcmailr.apiToken", apiToken);
         }
-        properties2.put("xcmailr.temporaryMailValidMinutes", String.valueOf(validMinutes));
-        writeMapToPropertiesFile(properties2, tempConfigFile2);
-        ConfigFactory.setProperty("xcmailr.temporaryConfigFile", "file:" + fileLocation);
+        writeProperty("xcmailr.temporaryMailValidMinutes", "3");
+        SendRequest.login(xcmailrEmail, xcmailrPassword);
     }
 
     @After
     public void deleteTempEmail() throws ClientProtocolException, IOException
     {
-        SendRequest.login(xcmailrEmail, xcmailrPassword);
         SendRequest.deleteTempEmail(tempEmail);
     }
 
     @Test
     public void testEmailCreated()
     {
         XcMailrApi.createTemporaryEmail(tempEmail);
-
-        new XcmailrLoginPage().login(xcmailrEmail,
-                                     xcmailrPassword)
-                              .openMailOverview().validateEmailCreated(tempEmail);
+        Assert.assertTrue(SendRequest.emailExists(tempEmail));
     }
 
     @Test
     public void testEmailExpired()
     {
+        writeProperty("xcmailr.temporaryMailValidMinutes", "1");
         XcMailrApi.createTemporaryEmail(tempEmail);
-        XcmailrOverviewPage mailOverview = new XcmailrLoginPage().login(xcmailrEmail, xcmailrPassword).openMailOverview();
-        mailOverview.validateEmailIsActive(tempEmail);
-        Selenide.sleep((validMinutes + 1) * 60000);
-        Selenide.refresh();
-        Selenide.sleep(1000);
-        mailOverview.validateEmailIsExpired(tempEmail);
+        Assert.assertFalse(SendRequest.emailExpired(tempEmail));
+        Selenide.sleep(120000);
```

Comment:
Add a note saying that the job for deleting e-mails only runs every minute so we have to wait 2 minutes to work reliably.